gef.py

```diff
@@ -8107,9 +8135,19 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
 
         addr = align_address(parse_address(args.location))
         size, fcode = self.SUPPORTED_SIZES[self.format]
+        values = args.values 
+
+        if size == 1: 
+            if values[0].startswith("$"):
+                var_name = values[0]
+                try:
+                    values = str(gdb.parse_and_eval(var_name)).replace("{", "").replace("}", "").replace(",","").split(" ")
```

Comment:
```suggestion
                    values = bytes(eval(gdb.parse_and_eval(var_name).format_string(raw=True)))
                    gef.memory.write(addr, values)
                    return
```

Reply:
this not works as expected, I made some changes to fix the problem:

try: 
                    values = bytes(eval(gdb.parse_and_eval(var_name).format_string(raw=True).replace("{", "").replace("}", "")))
                    print(values)
                    gef.memory.write(addr, values, length=len(values))
                    return

maybe this is not the pythonic way :\

Reply:
resolved in discord